Makefile

```diff
@@ -4,12 +4,12 @@ PHONIES += check download-and-verify help unpack unpack-clean
 GPG2      = gpg2
 WGET      = wget
 
-REPO      = measurement-kit/measurement-kit-jni-libs
+REPO      = measurement-kit/measurement-kit
 BASEURL   = https://github.com/$(REPO)/releases/download
 PACKAGE   = org/openobservatory/measurement_kit
 SRCDIR    = app/src/main
-TAG       =
-VERSION   = v0.1.2.2
+TAG       = -0
+VERSION   = v0.2.0-beta.2
```

Comment:
Uhm, the version tagged in here is not super up to date. I'm recompiling everything and pushing `0.2.7` compiled for Android such that we can use the real latest version.